File: 05 SWEA/D1/Java/src/Prob2070.java

/**
* title  : 2070. 큰 놈, 작은 놈, 같은 놈 (D1)
* time   : 101ms
* memory : 18364kb
*/

import java.io.*;
import java.util.*;
 
public class Prob2070 {
    
    public static void main(String[] args) throws IOException {
        
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int T = Integer.parseInt(br.readLine());
        for (int t = 1; t <= T; t++) {

            StringTokenizer st = new StringTokenizer(br.readLine());
            int n1 = Integer.parseInt(st.nextToken());
            int n2 = Integer.parseInt(st.nextToken());

            String answer = "";

            if (n1 == n2) {
                answer = "=";
            } else if (n1 < n2) {
                answer = "<";
            } else if (n1 > n2) {
                answer = ">";
            }

            sb.append("#" + t).append(" " + answer).append("\n");
        }

        System.out.println(sb);
    }
}

File: 05 SWEA/D1/Python/Prob2070.py

"""
title  : 2070. 큰 놈, 작은 놈, 같은 놈 (D1)
time   : 107ms
memory : 43488kb
"""

def solution():

    T = int(input())
    for tc in range(1, T+1):

        num1, num2 = map(int, input().split())

        answer = ""
        if num1 == num2:
            answer = "="
        elif num1 < num2:
            answer = "<"
        elif num1 > num2:
            answer = ">"
        
        print(f'#{tc} {answer}')

solution()